docs/creating-a-codelist-from-a-csv-file.md

```diff
@@ -9,11 +9,18 @@ and choose a file to upload from your hard drive.
 
         create-a-codelist-1.png
 
-Note that the CSV file should not have a header,
-and its first column must contain valid codes in the chosen coding system.
-(This limitation will be lifted.)
+The final codelist must be stored in CSV format to be imported 
```

Comment:
would probably add a note about OPCS codes close to the top and refer to bottom of page?